Makefile

```
@@ -112,6 +112,9 @@ geniso: $(KERNEL)
 genuefi:
 	$(shell bash $(BUILD_PREFIX)tools/grub.sh) -d /usr/lib/grub/x86_64-efi -o SayoriOS_UEFI.iso iso/ --locale-directory=/usr/share/locale/ -V "SayoriOS Soul"
 
+disk.img:
+	echo hello >disk.img
```

```suggestion
	fallocate -l 128M disk.img
```

This would be simpler